### ISSUE TITLE
Update Seurat release to "latest"

### ISSUE THREAD

**Issue by brianraymor** [chanzuckerberg/single-cell-data-portal#5532]
# Context

See [**#single-cell-four**](https://czi-sci.slack.com/archives/C03PXFMM7ST/p1692222741241779).

Seurat must be updated to 4.3.0 or its [latest release](https://github.com/satijalab/seurat/releases) at the last possible moment but prior to schema 4 ingestion testing in Dev.

**Comment by danieljhegeman** [comment]
It looks like the conversion is working by simply changing the SeuratObject version to 5.0.0

Here's a sample seurat file produced by the processing step with v5.0.0: `s3://env-rdev-datasets/pr-6130/05f80634-f06e-4353-a6c0-c4517dd854cd.rds`

**Comment by danieljhegeman** [comment]
[Dockerhub shows no sign](https://hub.docker.com/r/satijalab/seurat/tags) of a [Seurat 5 base image](https://github.com/chanzuckerberg/single-cell-data-portal/blob/6e9ab6abd636a30f27b79fde9aa0a26c75e6b75c/Dockerfile.processing_base#L1) from satijalab.

**Comment by danieljhegeman** [comment]
[The seurat Dockerfiles](https://github.com/satijalab/seurat-docker/tree/master) similarly have not been updated with one for Seurat 5...

**Comment by danieljhegeman** [comment]
@brianraymor I can _try_ manually building a base processing image with _SeuratObject v5_ which will still run with _Seurat 4_ under the hood...?

**Comment by brianraymor** [comment]
I defer to @nayib-jose-gloria to decide when he prefers to **cut bait** on Seurat v5 to reduce risk to the schedule. The new version can be addressed later in coordination with visium support in 2024 H1.

**Comment by nayib-jose-gloria** [comment]
@brianraymor @danieljhegeman the lack of a v5 beta dockerfile that we could even do exploratory work with does make me err on the side of postponing this work; we don't have a sense of what new dependencies are required to support v5, and I'd rather not postpone migration testing waiting on a release or get caught up trying to reverse-engineer the dependencies if a seurat 5 image is going to come out soon and we can stand to wait.

**Comment by danieljhegeman** [comment]
Ok. We'll set this work aside for now.

**Comment by danieljhegeman** [comment]
Noting that we do have some references to **Seurat v4** in the FE code that will also need to be changed.

**Comment by danieljhegeman** [comment]
Processing succeeds for schema 4 test dataset

**Comment by danieljhegeman** [comment]
Places with Seurat v4 text references that should be updated to v5:
-[docsite](https://cellxgene.dev.single-cell.czi.technology/docs/03__Download%20Published%20Data#:~:text=v0.8)%20and%20rds%20(-,Seurat%20v4,-)%20formats.%20All%20datasets)
-tooltip `"A .rds (Seurat v4) download is unavailable due to limitations in the R..."`
-[download radio button](https://github.com/chanzuckerberg/single-cell-data-portal/blob/7e3cccacf01bd615b3b2c647628996ca9195c360/frontend/src/components/Collections/components/Dataset/components/DownloadDataset/components/Content/components/DataFormat/index.tsx#L66)
-[copy caption](https://github.com/chanzuckerberg/single-cell-data-portal/blob/b63310675507b36b47083fe06f66ca383efdb350/frontend/src/components/Collections/components/Dataset/components/DownloadDataset/components/Content/components/DownloadLink/components/CopyCaption/index.tsx#L54)
cc @nayib-jose-gloria @brianraymor

**Comment by nayib-jose-gloria** [comment]
Thanks for tracking those down, Dan. Do you mind adding those changes to your PR? 

I'd also like to try the Seurat 5 conversion (+ confirming the Version is updated on the SeuratObject) on a couple of real datasets from prod before merging. I have a notebook snippet I'll send you on how to check using python

**Comment by danieljhegeman** [comment]
@nayib-jose-gloria to test the seurat file on his local machine

**Comment by danieljhegeman** [comment]
[Tracking conv](https://czi-sci.slack.com/archives/C03PXFMM7ST/p1699470480835219) with @jychien to verify a `.rds` file produced with our Seurat v5 processing image
